dataduct/steps/scripts/create_load_redshift_runner.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+"""
+Replacement for the load step to use the redshift COPY activity instead
```

Comment:
In general we only put them on the same line in case of docstrings inside a function. The file headers are mostly in the next line. But I will keep this in mind for the next commits.